src/app/modules/form/form.service.ts

```diff
@@ -343,3 +344,59 @@ export const retrievePublicFormsWithSmsVerification = (
     return okAsync(forms)
   })
 }
+
+export const createSampleSubmissionData = (
+  sampleData: Record<string, unknown>,
+  field: FormFieldDto,
+) => {
+  let sampleValue = null
+  let noOfOptions = 0
+  let randomSelectedOption = 0
+  switch (field.fieldType) {
+    case 'textarea':
+    case 'textfield':
+      sampleValue =
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+      break
+    case 'radiobutton':
```

Comment:
Just a thought, but are we going check for any of the validation stuffs in each field? 

for instance domain restriction in email, text min/max amount etc?

Reply:
that's true^ good point @foochifa, their processes might break if we just put in hard-coded values that are not within the values they want to set

maybe we can separate this into a separate issue and do it in a separate PR though? if not the scope of this PR might creep too much

Reply:
yes good point @foochifa ! I'll create a separate issue and PR for it

Reply:
#6335 - new issue